packages/node-cli/README.md

```diff
@@ -98,3 +98,5 @@ SuiteCloud CLI for Node.js is an open source project. Pull Requests are currentl
 
 ## [License](/LICENSE.txt)
 Copyright (c) 2021 Oracle and/or its affiliates The Universal Permissive License (UPL), Version 1.0.
+
+By installing this NPM package, you are accepting the installation of the SuiteCloud SDK dependency under this [license](https://www.oracle.com/downloads/licenses/oracle-free-license.html).
```

Comment:
I guess this need to be updated with Viktor's proposal of specifying the type of license.

Reply:
Yes, @antoniofruci, final version should be: By installing this NPM package, you are accepting the installation of the SuiteCloud SDK dependency under the [Oracle Free Use Terms and Conditions license] (https://www.oracle.com/downloads/licenses/oracle-free-license.html).